demisto_sdk/commands/init/contribution_converter.py

```diff
@@ -466,20 +466,26 @@ def convert_contribution_to_pack(self, files_to_source_mapping: Dict = None):
                 )
 
     @staticmethod
-    def extract_pack_version(script):
+    def extract_pack_version(script_type, script):
         """
         extract the pack version from script if exists, returns 0.0.0 if version was not found.
         """
         if script:
             try:
-                pack_version_reg = re.search(
-                    r"### pack version: (\d+\.\d+\.\d+)", script
-                )
+                pack_version_reg = None
+                # script type can be python2 or python3, javascript and powersell.
+                if "python" in script_type or script_type == "powershell":
+                    pack_version_reg = re.search(
+                        r"### pack version: (\d+\.\d+\.\d+)", script
+                    )
+                elif script_type == "javascript":
+                    pack_version_reg = re.search(
+                        r"// pack version: (\d+\.\d+\.\d+)", script
+                    )
```

Comment:
can we use a single regex, catching both, instead of adding the condition here? e.g. `(?:###|//) pack version: (\d+\.\d+\.\d+)`